docs/DEPLOY.md

```diff
@@ -0,0 +1,19 @@
+# Deploy
+
+##### `sls deploy`
+- Description: Deploy resource group containing function app and other resources. Also deploys Function App code
+- Two Approaches:
+  - Deploy resource group, upload packaged artifact directly to function app. Sets function app `RUN_FROM_PACKAGE` setting to `1`.
+  - Deploy resource group, upload packaged function app to blob storage with version name. Sets function app `RUN_FROM_PACKAGE` setting to path of zipped artifact in blob storage
+- Open Questions:
+  - Default to deploy to blob storage? Default to zip deploy directly to function app?
+  - Container name for code artifacts? Hard coded? Configurable? Default to ARTIFACTS?
```

Comment:
`ARTIFACTS` default make sense, but also provide customization in `serverless.yml`

Reply:
Does serverless let you specify the name of the S3 bucket?  My hunch is no... that being said, I don't have a huge preference one way or the other.